src/message_pool.zig

```diff
@@ -46,14 +50,13 @@ pub const MessagePool = struct {
             .free_list = null,
             .header_only_free_list = null,
         };
-
-        var i: usize = 0;
-        while (i < config.connections_max) : (i += 1) {
-            {
+        {
+            var i: usize = 0;
+            while (i < 2 * config.connections_max) : (i += 1) {
```

Comment:
What do you think of exposing these in `config.zig` with:

```
const message_bus_messages_max = config.connections_max * 2;
const message_bus_headers_max = config.connections_max * 16; // We can have more of these since they're small. 
```

Reply:
Makes sense to me :+1: